Load icon from non-scalable path

razzeee opened the pull request:
The scalable paths should expect a svg icon, but we only have a .png so we need to load it from a non-scalable path

I had to transform the icon slightly as the one from the repo wasn't a square. 

Do you have the source files for this image saved somewhere or should we backup the higher resolution image from before somewhere?

Might help with #421

ImDevinC commented on src/tray.py:
```diff
@@ -17,8 +20,9 @@ def __init__(self):
-        self.set_icon("com.core447.StreamController", os.path.join(gl.top_level_dir, "Assets", "icons", "hicolor", "scalable", "apps"))
+        super().__init__(
+            self.menu, self.MenuPath, self.IndicatorPath, self.AppId, "StreamController"
+        )
```
```suggestion
        )
        self.set_icon("com.core447.StreamController")
```

ImDevinC replied:
We can use the app name to set the icon now that we have proper icons in place. Without this change, the icon just loads as a green alien icon (or some other default)

ImDevinC replied:
Confirmed that with this change the flatpak loads the icon properly in both Gnome with Appindicator extension installed, and on Hyprland with a tray icon extension installed.

razzeee committed:
Load icon from non-scalable path

The scalable paths should expect a svg icon, but we only have a .png so we need to load it from a non-scalable path
